rust/core-lib/src/tabs.rs

```diff
@@ -678,8 +679,9 @@ impl CoreState {
         //TODO: currently we only use the file's modification time when
         // determining if a file has been changed by another process.
         // A more robust solution would also hash the file's contents.
+        let tail_toggle = self.file_manager.tail_toggle;
 
-        if has_changes && is_pristine {
+        if has_changes && is_pristine && tail_toggle {
```

Comment:
This should be called even if `tail_toggle` is false (and `tail_toggle` should probably be a per-file setting).

Reply:
This is clear to me now. I will bring back `reload`.